humans/templates/shipanaro/base.html

```diff
@@ -1,10 +1,10 @@
 {% extends 'shipanaro/base.html' %}
 {% load i18n static %}
 {% block extrastyle %}
-<link
+<!--link
   href="https://fonts.googleapis.com/css?family=Open+Sans"
   rel="stylesheet"
-/>
+/-->
```

Comment:
Arguable, but this speeds up page loads and I don't much see the benefit of this font :sweat_smile: